### DISCUSSION TITLE
제출한 코드가 여러 개일 때 런타임이 가장 긴 코드가 업로드 되는 상황

### DISCUSSION THREAD

**Discussion by jinny-l** [BaekjoonHub/BaekjoonHub#140] [Q&A] [answered]
안녕하세요, 코린이입니다.
깃허브 사용이 익숙하지 않아 혹시 known issue이거나
제가 잘못 이해한 것일 수도 있어 미리 양해 부탁드립니다. ;ㅁ;

velog에 작성해주신 설명법을 보고, 제출한 코드가 여러개일 경우 
런타임 > 메모리 > 코드 길이 순으로 고려하여 최고 성능의 코드만 업로드된다는 것을 확인하였는데요,
이전에 풀었던 코드 업로드 시,  런타임이 제일 길고 메모리도 제일 많이 잡아먹은 코드가 업로드 되어
혹시 버그일지 아니면 제가 잘못 이해한 것일지 몰라 찾아오게 되었습니다.

![스크린샷 2022-09-26 오전 1 23 33](https://user-images.githubusercontent.com/108214590/192154663-f7eb581a-2cc5-498f-aa85-3325640b1d2f.png)

**Comment by zosungwoo** [comment, marked as answer]
Resolved in PR #208

**Comment by getCurrentThread** [comment]
현재 조치가 된 문제이나, 추가 문제가 있어서 현재 업데이트 지연 중에 있습니다.
